### PR TITLE
Updated UITextFieldExtensions

### PR DESCRIPTION
**Pull request by EdiLT**
### PR's key points
Moved and optimized older UITextFieldView extensions
 
### How to review this PR?
Check naming, functionality and tests for this extensions

### CODE REVIEW

**Comment by sebastianvarela** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          6 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false"><strong>📢 [iOS]</strong> Executed 97 tests, with 0 failures (0 unexpected) in 1.495 (2.149) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false"><strong>📢 [mac]</strong> Executed 61 tests, with 0 failures (0 unexpected) in 0.780 (0.943) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false"><strong>📢 [tv]</strong> Executed 97 tests, with 0 failures (0 unexpected) in 0.911 (1.072) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">⌛️ <strong>[iOS]</strong> Slowest test: test_html_value() took 0.836 seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">⌛️ <strong>[mac]</strong> Slowest test: test_html_value() took 0.454 seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">⌛️ <strong>[tv]</strong> Slowest test: test_html_value() took 0.411 seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Tests/Extensions/UIKit/UIViewExtensionsTests.swift [hunk 1]
```diff
@@ -4,6 +4,36 @@ import UIKit
 
 final class UIViewExtensionsTests: XCTestCase {
 
+    func testBorderColor() {
```
Comment (r-pedraza) [inline Tests/Extensions/UIKit/UIViewExtensionsTests.swift]: The naming of tests isn't camel case, It,s should contained _

File: Tests/Extensions/UIKit/UIViewExtensionsTests.swift [hunk 2]
```diff
@@ -4,6 +4,36 @@ import UIKit
 
 final class UIViewExtensionsTests: XCTestCase {
```
Comment (r-pedraza) [inline Tests/Extensions/UIKit/UIViewExtensionsTests.swift]: The naming of tests isn't camel case, It,s should contained _

File: Tests/Extensions/UIKit/UIViewExtensionsTests.swift (hunk 1 above)
Comment (sebastianvarela) [inline Tests/Extensions/UIKit/UIViewExtensionsTests.swift]:
```suggestion
    func test_border_color() {
```

File: Tests/Extensions/UIKit/UIViewExtensionsTests.swift [hunk 3]
```diff
@@ -4,6 +4,36 @@ import UIKit
...
+        let view = UIView(frame: frame)
+        view.borderColor = nil
+        XCTAssertNil(view.borderColor)
+        view.borderColor = UIColor.blue
+        XCTAssertNotNil(view.layer.borderColor)
+        XCTAssertEqual(view.borderColor, UIColor.blue)
+        XCTAssertEqual(view.layer.borderColor, UIColor.blue.cgColor)
+    }
+
+    func testBorderWidth() {
```
Comment (sebastianvarela) [inline Tests/Extensions/UIKit/UIViewExtensionsTests.swift]:
```suggestion
    func test_border_width() {
```

File: Tests/Extensions/UIKit/UIViewExtensionsTests.swift [hunk 4]
```diff
@@ -4,6 +4,36 @@ import UIKit
...
+        let frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+        let view = UIView(frame: frame)
+        view.borderWidth = 1
+        XCTAssertEqual(view.layer.borderWidth, 1)
+
+        view.borderWidth = 0
+        XCTAssertEqual(view.borderWidth, 0)
+    }
+
+    func testCornerRadius() {
```
Comment (sebastianvarela) [inline Tests/Extensions/UIKit/UIViewExtensionsTests.swift]:
```suggestion
    func test_corner_radius() {
```

File: Tests/Extensions/UIKit/UIViewExtensionsTests.swift [hunk 5]
```diff
@@ -4,6 +4,36 @@ import UIKit
 
 final class UIViewExtensionsTests: XCTestCase {
```
Comment (sebastianvarela) [inline Tests/Extensions/UIKit/UIViewExtensionsTests.swift]:
```suggestion
class UIViewExtensionsTests: XCTestCase {
```

**Pull request merged**